### DISCUSSION TITLE
Welcome to cuckoo Discussions!

### DISCUSSION THREAD

**Discussion by tromp** [tromp/cuckoo#106] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by davhello** [comment]
hi tromp,

i post here too just in case of simplier.
what can manage your code to find proofsize cycles on same compute?
I am not familiar to cuckoo, i did some tests by managing trims number but without good result.
So actually i use default parameters.

do you have an advise to manage that? do i have to retry? and make a loop?

Thanks for your time

**Reply by davhello** [reply]
i found the solution inside closed points :
FLUSHA=16 and FLUSHB=8
to 
FLUSHA=32 and FLUSHB=16

regards

**Reply by tromp** [reply]
> what can manage your code to find proofsize cycles on same compute?

I don't understand the question.  By same compute, do you mean on the same graph?
It will compute multiple solutions if there are multiple 42-cycles. Which is relatively rare (about 1 in 42^2 graphs).

**Reply by davhello** [reply]
lol i understand your question, i just did that to bench and find why sometimes i was finding some proof and some no on same values.
That wont be like that in reality.
By same compute i meant same header and same nonce.
But it was just because i saw differences between 2 same compute.

Now it is better about 996/1000 found proof on same calcul. which is still strange but ok that will do the job.

**Reply by ampholsps37-crypto** [reply]
Hello
